lib/Installation/Partitioner/FormattingOptionsPage.pm

```diff
@@ -85,10 +85,8 @@ sub select_filesystem {
     return if $skip;
     assert_screen(FORMATTING_OPTIONS_PAGE);
     send_key($self->{filesystem_shortcut});
-    wait_screen_change(sub {
```

Comment:
`wait_screen_change` is there for some reason. not sure why you need to delete it.

Reply:
you are right. But my mistake is somewhere else. the indention was not to remove the `wait_screen_change` but to use the internal. i will fix

Reply:
Code looks better now, but does it have any functional change? I'm just curios =)

Reply:
based on the documentation no. Actually i would trust more passing this to the isovideo. I am waiting results from the VRs. Although i dont think we have improve anything significant here.

Reply:
Yeah, main help should come from the L89, so that we will match FS before pressing up key one time too much.